monix-eval/shared/src/main/scala/monix/eval/TaskLocal.scala

```diff
@@ -116,7 +124,7 @@ final class TaskLocal[A] private (default: => A) {
     *   // Should yield 200 on execution, regardless of what value
     *   // we have in `local` at the time of evaluation
     *   val task = local.bind(100)(Task {
-    *     local.get * 2
+    *     local.read * 2
```

Comment:
Actually, `local` here is `Task[TaskLocal[Int]]` and `local.read` returns a `Task[Int]` too, so it cannot be multiplied directly. So the code sample should be something along the lines of

```scala
for {
  local <- TaskLocal(0)
  value <- local.bind(100)(local.read.map(_ * 2))
} yield value
```